xt/01.2-deps.t

```diff
@@ -22,6 +22,7 @@ push @on_disk, 'bin/ledgersmb-server.psgi';
 ok_dependencies($file, \@on_disk,
                 phases => 'runtime',
                 ignores => [
+                    'File::Grep', # used to make PO from JSON
```

Comment:
If you specify `File::Grep` as a `devel` dependency, this command won't go search for it (that's what the `'runtime'` is for).

Reply:
Fixed